Crawlers/twitter_accounts/adaderana.py

```diff
@@ -0,0 +1,91 @@
+import tweepy
+from elasticsearch import Elasticsearch, helpers
+from threading import Thread, Timer
+from datetime import datetime
```

Comment:
Have these libraries been added to requirements.txt ? Tweepy was not there.

Reply:
Added it.